src/applications/disability-benefits/781/helpers.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export const introductionText = (
+  <div>
+    <p>
+      We'll now ask you questions about the stressful event or events related to your PTSD. We understand that some of the questions maybe difficult to answer. The information you provide here will help us understand you situation and research your claim.
```

Comment:
There's a misspelling here. "...help us understand you situation..." should be "...help us understand your situation..."